pkg/elm/validators.go

```diff
@@ -171,32 +171,15 @@ func PresenceValidator(isOptional bool, isRepeated bool, validatorOptions Valida
 			FieldValidator:      validatorOptions.CustomValidator,
 			SubMessageValidator: subMessageValidator,
 		}
-	} else if validatorOptions.RequiredIfFieldTrue != nil {
-		for _, field := range parentMessage.GetField() {
-			if field.GetName() == *validatorOptions.RequiredIfFieldTrue {
-				otherField := stringextras.LowerCamelCase(*validatorOptions.RequiredIfFieldTrue)
-				if IsOptional(field) || field.OneofIndex != nil {
-					return Validator{
-						Package:             ValidatorPackageName(),
-						MessageValidator:    "requiredIfMaybeFieldTrue",
-						RequiredIfFieldTrue: &otherField,
-						ModelValidator:      validatorOptions.CustomModelValidator,
-						FieldValidator:      validatorOptions.CustomValidator,
-						SubMessageValidator: subMessageValidator,
-					}
-				} else {
-					return Validator{
-						Package:             ValidatorPackageName(),
-						MessageValidator:    "requiredIfFieldTrue",
-						RequiredIfFieldTrue: &otherField,
-						ModelValidator:      validatorOptions.CustomModelValidator,
-						FieldValidator:      validatorOptions.CustomValidator,
-						SubMessageValidator: subMessageValidator,
-					}
-				}
-			}
+	} else if validatorOptions.RequiredIf != nil {
+		return Validator{
+			Package:             ValidatorPackageName(),
+			MessageValidator:    "requiredIf CustomValidators." + *validatorOptions.RequiredIf,
```

Comment:
This bit is tricky.  We reuse `validators.go` over in ruby world.  If we instead leave it in `RequiredIf`, we can use different namespacing syntax in the ruby + elm generators.

The ruby version transforms this struct into a string here:
https://github.com/thematthopkins/protoc-gen-rbi/blob/ad15fc0a2aed0a62d85fe0c066c9c793a4938751/ruby_types/ruby_types.go#L60

Once this PR merged in then, use this command to have the ruby generator reference the latest version of this package:
https://github.com/thematthopkins/protoc-gen-rbi/blob/ad15fc0a2aed0a62d85fe0c066c9c793a4938751/run-all-tests.sh#L10

At some point, we may be best best off importing all this into the loadpay repo.